extensions/azure/storage/AzureBlobStorage.h

```diff
@@ -24,33 +24,29 @@
 #include <string>
 #include <vector>
 
-#include "BlobStorage.h"
+#include "BlobStorageClient.h"
 #include "azure/storage/blobs.hpp"
 #include "core/logging/Logger.h"
 #include "core/logging/LoggerConfiguration.h"
+#include "gsl/gsl-lite.hpp"
```

Comment:
When introducing gsl-lite to the codebase, I created `"utils/gsl.h"` with the intention of it being the common entry point from minifi to gsl, including a namespace alias of `::org::apache::nifi::minifi::gsl` to `::gsl_lite`, following [gsl-lite's own guidelines for libraries](https://github.com/gsl-lite/gsl-lite#using-gsl-lite-in-libraries). I also introduced one gsl-related utility in the same file, but I'm open to moving that elsewhere.

Consider using `"utils/gsl.h"` and the namespace alias to access gsl-lite functionality.

Reply:
Thanks, I didn't know about this purpose of the header, I replaced the old ones in 1ff3b235bc9a137a979a98f4c8bf3c9565676a39